Contents/Perks/ModPerk.cs

```diff
@@ -341,17 +341,27 @@ public class BlessingOfVortex : Perk {
 			Tooltip =
 				"+ 78% increased odds for range" +
 				"\n+ 5% range critical strike chance" +
-				"\n+ 25% range critical damage";
+				"\nYou have 0.2% chance to deal 4x damage";
 			StackLimit = 5;
 		}
+		public override string ModifyToolTip() {
+			if (StackAmount > 0)
+				return "+ 78% increased odds for range" +
+				"\n+ 5% range critical strike chance" +
+				"\nIncrease chance to deal 4x damage by 1%";
+			return
+				"+ 78% increased odds for range" +
+				"\n+ 5% range critical strike chance" +
+				"\nYou have 1% chance to deal 4x damage";
```

Comment:
Why spaces between `+` and `%`? Kinda odd.

(TODO: Prefer using localization file instead of hard-coding text like that; in future:tm:)